web/src/main/java/com/futurewei/alcor/web/entity/route/RouterUpdateInfo.java

```diff
@@ -29,8 +29,8 @@
     @JsonProperty("operation_type")
     private String operationType;
 
-    @JsonProperty("old_subnet_ids")
-    private List<String> oldSubnetIds;
+    @JsonProperty("gateway_port_ids")
```

Comment:
@kevin-zhonghao @cj-chung  FYI. Please review this change as well.

Reply:
@chenpiaoping  This entity seems only used by PM's internal logic. It's a return object from updateL3Neighbors function and consumable by PM only.